herbert_core/classes/MPIFramework/@MessagesFilebased/private/init_framework_.m

```diff
@@ -23,8 +23,8 @@
     end
 elseif(is_string(framework_info))
     % use process ID and time as job ID. This prevents clashes between processes
-    obj.job_id_ = sprintf('%05i', feature('getpid'), ...
-                          round(datetime('now').Second*1e3));
+    obj.job_id_ = sprintf('%s_%08i',framework_info,...
```

Comment:
this job id generation logic is duplicated in `iMessagesFramework` so should be changed to there to be consistent

Reply:
fixed in the next commit